src/main/scala/diplomacy/Nodes.scala

```diff
@@ -411,11 +419,11 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   private var bundlesSafeNow = false
   // Accessors to the result of negotiation to be used in LazyModuleImp:
   def out: Seq[(BO, EO)] = {
-    require(bundlesSafeNow, s"${name}.out should only be called from the context of its module implementation")
+    require(bundlesSafeNow, s"${name}.out should only be called from the context of its LazyModule(${scope.get.name}) implementation, currently (${LazyModule.scope})")
```

Comment:
Let's change this to `${name}.out should only be called from within the context of ${scope.get.name}'s LazyModuleImp, but the current scope is ${LazyModule.scope}`